src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CreateCaseCallbackServiceTest.java

```diff
@@ -78,19 +60,13 @@
     private ServiceConfigProvider serviceConfigProvider;
 
     @Mock
-    private TransformationClient transformationClient;
-
-    @Mock
-    private AuthTokenGenerator s2sTokenGenerator;
-
-    @Mock
-    private CoreCaseDataApi coreCaseDataApi;
+    private CcdApi ccdApi;
 
     @Mock
-    private CcdApi ccdApi;
+    private CcdNewCaseCreator ccdNewCaseCreator;
 
     @Mock
-    private PaymentsProcessor paymentsProcessor;
+    private ExceptionRecordFinalizer exceptionRecordFinalizer;
```

Comment:
Now that we have this ExceptionRecordFinalizer (which is a good idea), we should verify that it was called properly and also that we return its result.

Reply:
Will do